connector/connect/doc/client_connection_string.md

```diff
@@ -0,0 +1,110 @@
+# Connecting to Spark Connect using Clients
```

Comment:
The usage documentation should better be in https://github.com/apache/spark/tree/master/docs as an md file so it can be properly documented.